### PR TITLE
doc: mention copying keyrings and adjust node names in manual deployment example

### PR DESCRIPTION
**Pull request by rokj**

### CODE REVIEW

**Comment by rokj** [comment]
Hi, what is problem Docs: build check?

**Comment by tchaikov** [comment]
jenkins test docs

**Comment by rokj** [comment]
Yes, but it seems jenkins test fails because it failed to get some package.

**Comment by idryomov** [comment]
jenkins test docs

File: doc/install/manual-deployment.rst [hunk 1]
```diff
@@ -308,25 +308,29 @@ bluestore
 ^^^^^^^^^
```
Comment (idryomov) [inline doc/install/manual-deployment.rst:308]: A couple of now obsolete references to `node2` and `node3` above this line.

File: doc/install/manual-deployment.rst [hunk 2]
```diff
@@ -308,25 +308,29 @@ bluestore
...
-	ssh {node-name}
+	copy /var/lib/ceph/bootstrap-osd/ceph.keyring from monitor node (mon-node1) to /var/lib/ceph/bootstrap-osd/ceph.keyring on osd node (osd-node1)
+	ssh {osd node}
 	sudo ceph-volume lvm create --data {data-path}
 
    For example::
 
-	ssh node1
+    	ssh mon-node1
+    	scp /var/lib/ceph/bootstrap-osd/ceph.keyring root@osd-node1:/var/lib/ceph/bootstrap-osd/ceph.keyring
```
Comment (idryomov) [inline doc/install/manual-deployment.rst:318]: Nit: This presupposes that `mon-node1` has ssh credentials for `osd-node1` (and, by extension, other OSD nodes).  I would provide an example using `scp -3` (remote to remote through local "admin" node).

**Review by rokj — COMMENTED** [review]
I've included requested changes.

File: doc/install/manual-deployment.rst [hunk 3]
```diff
@@ -301,32 +301,35 @@ for use with Ceph. The ``ceph-volume`` utility creates the OSD ID by
...
 
-	ssh {node-name}
+	copy /var/lib/ceph/bootstrap-osd/ceph.keyring from monitor node (mon-node1) to /var/lib/ceph/bootstrap-osd/ceph.keyring on osd node (osd-node1)
+	ssh {osd node}
 	sudo ceph-volume lvm create --data {data-path}
 
    For example::
 
-	ssh node1
+    	ssh mon-node1
```
Comment (idryomov) [inline doc/install/manual-deployment.rst:316]: One shouldn't need to ssh to `mon-node1` for `scp -3` copy.

**Comment by idryomov** [comment]
Please squash these additional commits into the first commit and sign it per https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#sign-your-work.

**Comment by rokj** [comment]
Did squashed commit and sign it.

**Review by idryomov — APPROVED** [review]

**Pull request merged**